src/transformers/utils/generic.py

```diff
@@ -867,3 +869,48 @@ class LossKwargs(TypedDict, total=False):
     """
 
     num_items_in_batch: Optional[int]
+
+
+def is_timm_hub_checkpoint(pretrained_model_name_or_path: str) -> bool:
+    "Checks whether a checkpoint is a hub model hosted under the timm organization."
+    if not isinstance(pretrained_model_name_or_path, str):
+        return False
+
+    if os.path.isfile(pretrained_model_name_or_path) or os.path.isdir(pretrained_model_name_or_path):
+        return False
+
```

Comment:
and I'd therefore remove this

Reply:
function removed with https://github.com/huggingface/transformers/commit/ff6efde8d0d77baa83e9e7a264aec84618751d1b